projects/plugins/jetpack/extensions/blocks/map/component.js

```diff
@@ -306,20 +307,33 @@ export class Map extends Component {
 		}
 		this.initMap( mapCenter );
 	};
+
 	loadMapLibraries() {
 		const { apiKey } = this.props;
-		Promise.all( [
-			import( /* webpackChunkName: "map/mapbox-gl" */ 'mapbox-gl' ),
-			import( /* webpackChunkName: "map/mapbox-gl" */ 'mapbox-gl/dist/mapbox-gl.css' ),
```

Comment:
Note that by default [Webpack dynamic imports](https://webpack.js.org/api/module-methods/#magic-comments) are `lazy`, i.e. loaded asynchronously. Just noting in case you wanna ensure the same loading method with `<script async>`?

Reply:
Thanks. With the way this is currently setup the script tag is not injected into the iframe header until after the page has loaded so adding async may not make any difference, but will do some double checking on that.